test/index.js

```diff
@@ -462,6 +461,27 @@ test('catching errors with a json payload', function (t) {
     service._abortPaging.restore()
     t.end()
   })
+})
+
+test('logging with a passed in logger', function (t) {
+  var logger = {}
+  logger.log = function (level, message) {
+    t.equal(level, 'test')
+    t.equal(message, 'test')
+    t.end()
+  }
+  var service = new FeatureService('htt://service.com/mapserver/3', {logger: logger})
+  service.log('test', 'test')
```

Comment:
I think in a case like this, if logger.log never gets called the test is still passing. This is a good place to use `t.plan(2)` at the top of the test block instead of `t.end()`.